### DISCUSSION TITLE
Thumbnail image from DNG file is inappropriate

### DISCUSSION THREAD

**Discussion by SAN2INGIT** [ImageMagick/ImageMagick#5085] [Help]
I have a DNG [image ]( https://filesamples.com/samples/image/dng/sample1.dng) and a cropped monochromatic version of the image. Both are generating the same file as thumbnail when I run any of the below commands:
`magick.exe C:\sample1crop.dng -resize 500x375 C:\crop-T.JPG`
or 
`magick.exe dng:C:\sample1crop.dng -intent relative -sample 500x375> -strip -auto-orient -density 72 C:\crop-T.JPG`
or
`magick.exe convert dng:C:\sample1original.dng -thumbnail 500x375 -filter -auto-orient -density 72 C:\orig-T.JPG`

Te DNG files link: https://spaces.hightail.com/space/ThDEDYZVey 

The below two images are generated from the above cmds, and as seen in the images, for all the commands, the resulting images are the same.

![orig-T](https://user-images.githubusercontent.com/32328926/165252464-68244908-ad1c-49dd-83d2-f76ce4d19b3a.JPG)
Original image thumbnail

![crop-T](https://user-images.githubusercontent.com/32328926/165252449-c1f54077-790e-4b12-8125-b31baf6ab95c.JPG)
Cropped image thumbnail

**Comment by snibgo** [comment]
I'm not sure what you are asking. You show 3 commands and 2 images.

The images are the same size, but different. The second is sharper than the first. They have an RMSE of 2%, and this difference can be easily seen. Neither is a crop of the other.

**Reply by SAN2INGIT** [reply]
Apologies if my query was not clear, both the attached images are thumbnails, that I got for running both the original and cropped version of the DNG file. As you mentioned for both the cases the resulting image is same, not cropped or monochrome, like the original DNG file.

**Comment by snibgo** [comment]
Your IM commands do not crop images, or make them monochrome.

You haven't uploaded sample1crop.dng or sample1original.dng, so I can't comment on them. How did you make the "cropped" version? How do you know it is cropped? How do you know that either is monochrome? Perhaps the pixel data has not changed, but metadata says "this image should be croped, and made monochrome."

**Reply by SAN2INGIT** [reply]
The IM commands are for generating the thumbnail, they were applied on two different DNG images, one of which is a cropped image of the other.
While I am trying to upload the DNG images as a zip file, mentioning below the metadata for both the files from exif tool in case if that helps. The metadata has it mentioned as well about the image cropping
**sample1original.dng:**
[{
  "SourceFile": "C:/86854/SLS/Issues/ART-73712/sample1original.dng",
  "ExifTool": {
    "ExifToolVersion": 12.29,
    "Warning": "[Minor] Not decoding some large array(s). Ignore minor errors to decode"
  },
  "File:System": {
    "FileName": "sample1original.dng",
    "Directory": "C:/86854/SLS/Issues/ART-73712",
    "FileSize": "6.1 MiB",
    "FileModifyDate": "2022:04:25 01:57:44+05:30",
    "FileAccessDate": "2022:04:25 01:57:44+05:30",
    "FileCreateDate": "2022:04:25 01:57:44+05:30",
    "FilePermissions": "-rw-rw-rw-"
  },
  "File": {
    "FileType": "DNG",
    "FileTypeExtension": "dng",
    "MIMEType": "image/x-adobe-dng",
    "ExifByteOrder": "Little-endian (Intel, II)"
  },
  "EXIF:IFD0": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 256,
    "ImageHeight": 171,
    "BitsPerSample": "8 8 8",
    "Compression": "Uncompressed",
    "PhotometricInterpretation": "RGB",
    "Make": "Canon",
    "Model": "Canon EOS 350D DIGITAL",
    "StripOffsets": 6241370,
    "Orientation": "Horizontal (normal)",
    "SamplesPerPixel": 3,
    "RowsPerStrip": 171,
    "StripByteCounts": 131328,
    "PlanarConfiguration": "Chunky",
    "ModifyDate": "2009:04:22 02:03:33",
    "Artist": "unknown",
    "TimeZoneOffset": "-4 -4",
    "ImageNumber": 176,
    "DNGVersion": "1.2.0.0",
    "DNGBackwardVersion": "1.1.0.0",
    "UniqueCameraModel": "Canon EOS 350D",
    "LocalizedCameraModel": "Canon EOS 350D Digital",
    "ColorMatrix1": "0.6159 -0.0945 -0.0745 -0.6846 1.3563 0.3684 -0.0802 0.1086 0.7555",
    "ColorMatrix2": "0.6018 -0.0617 -0.0965 -0.8645 1.5881 0.2975 -0.153 0.1719 0.7642",
    "CameraCalibration1": "1.0491 0 0 0 1 0 0 0 0.9973",
    "CameraCalibration2": "1.0491 0 0 0 1 0 0 0 0.9973",
    "AnalogBalance": "1 1 1",
    "AsShotNeutral": "0.444396 1 0.669514",
    "BaselineExposure": 0.25,
    "BaselineNoise": 1.33,
    "BaselineSharpness": 1.5,
    "LinearResponseLimit": 1,
    "CameraSerialNumber": 830550471,
    "DNGLensInfo": "10-22mm f/?",
    "ShadowScale": 1,
    "CalibrationIlluminant1": "Standard Light A",
    "CalibrationIlluminant2": "D65",
    "RawDataUniqueID": "157A7EF208EDE0942DE6A8D6C0D05FB0",
    "OriginalRawFileName": "_MG_2224.CR2",
    "CameraCalibrationSig": "com.adobe",
    "ProfileCalibrationSig": "com.adobe",
    "ProfileName": "ACR 4.4",
    "ProfileEmbedPolicy": "Allow Copying",
    "ProfileCopyright": "Copyright 2008 Adobe Systems, Inc.",
    "ForwardMatrix1": "0.8255 -0.0022 0.1411 0.424 0.6455 -0.0695 0.0908 -0.234 0.9683",
    "ForwardMatrix2": "0.7623 0.0785 0.1235 0.3987 0.7409 -0.1396 0.0431 -0.1074 0.8894",
    "PreviewApplicationName": "Adobe Photoshop Lightroom",
    "PreviewApplicationVersion": 2.1,
    "PreviewSettingsDigest": "27f75e2e04a9540b92f6c2594981bc13",
    "PreviewColorSpace": "sRGB",
    "PreviewDateTime": "2008:12:14 17:54:20-04:00",
    "RawImageDigest": "d016d66e9a31680a1bca9b61799412f1",
    "ThumbnailTIFF": "(Binary data 131544 bytes, use -b option to extract)"
  },
  "EXIF:SubIFD": {
    "SubfileType": "Full-resolution image",
    "ImageWidth": 3516,
    "ImageHeight": 2328,
    "BitsPerSample": 16,
    "Compression": "JPEG",
    "PhotometricInterpretation": "Color Filter Array",
    "SamplesPerPixel": 1,
    "PlanarConfiguration": "Chunky",
    "TileWidth": 256,
    "TileLength": 240,
    "TileOffsets": "(Binary data 1096 bytes, use -b option to extract)",
    "TileByteCounts": "(Binary data 839 bytes, use -b option to extract)",
    "CFARepeatPatternDim": "2 2",
    "CFAPattern2": "0 1 1 2",
    "CFAPlaneColor": "Red,Green,Blue",
    "CFALayout": "Rectangular",
    "BlackLevelRepeatDim": "1 1",
    "BlackLevel": 0,
    "BlackLevelDeltaV": "(large array of 2314 rational64s values)",
    "WhiteLevel": 4095,
    "DefaultScale": "1 1",
    "DefaultCropOrigin": "10 5",
    "DefaultCropSize": "3456 2304",
    "BayerGreenSplit": 0,
    "AntiAliasStrength": 1,
    "BestQualityScale": 1,
    "ActiveArea": "14 42 2328 3516",
    "MaskedAreas": "14 0 2328 40"
  },
  "EXIF:SubIFD1": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 1024,
    "ImageHeight": 683,
    "BitsPerSample": "8 8 8",
    "Compression": "JPEG",
    "PhotometricInterpretation": "YCbCr",
    "PreviewImageStart": 6150264,
    "SamplesPerPixel": 3,
    "RowsPerStrip": 683,
    "PreviewImageLength": 90751,
    "PlanarConfiguration": "Chunky",
    "YCbCrCoefficients": "0.299 0.587 0.114",
    "YCbCrSubSampling": "YCbCr4:2:0 (2 2)",
    "YCbCrPositioning": "Co-sited",
    "ReferenceBlackWhite": "0 255 128 255 128 255",
    "PreviewApplicationName": "Adobe Photoshop Lightroom",
    "PreviewApplicationVersion": 2.1,
    "PreviewSettingsDigest": "27f75e2e04a9540b92f6c2594981bc13",
    "PreviewColorSpace": "sRGB",
    "PreviewDateTime": "2008:12:14 17:54:20-04:00",
    "PreviewImage": "(Binary data 90751 bytes, use -b option to extract)"
  },
  "EXIF:ExifIFD": {
    "ExposureTime": "1/800",
    "FNumber": 7.1,
    "ExposureProgram": "Manual",
    "ISO": 100,
    "ExifVersion": "0221",
    "DateTimeOriginal": "2008:12:14 15:54:54",
    "CreateDate": "2008:12:14 15:54:54",
    "ComponentsConfiguration": "Y, Cb, Cr, -",
    "ShutterSpeedValue": "1/800",
    "ApertureValue": 7.1,
    "ExposureCompensation": 0,
    "MaxApertureValue": 4.0,
    "MeteringMode": "Multi-segment",
    "Flash": "Off, Did not fire",
    "FocalLength": "20.0 mm",
    "FlashpixVersion": "0100",
    "ColorSpace": "Uncalibrated",
    "CustomRendered": "Normal",
    "ExposureMode": "Auto bracket",
    "WhiteBalance": "Auto",
    "SceneCaptureType": "Standard"
  },
  "Composite": {
    "Aperture": 7.1,
    "CFAPattern": "[Red,Green][Green,Blue]",
    "ImageSize": "3516x2328",
    "Megapixels": 8.2,
    "ShutterSpeed": "1/800",
    "FocalLength35efl": "20.0 mm",
    "LightValue": 15.3
  }
}]



**sample1crop.dng**
[{
  "SourceFile": "C:/86854/SLS/Issues/ART-73712/sample1crop.dng",
  "ExifTool": {
    "ExifToolVersion": 12.29,
    "Warning": "[Minor] Not decoding some large array(s). Ignore minor errors to decode"
  },
  "File:System": {
    "FileName": "sample1crop.dng",
    "Directory": "C:/86854/SLS/Issues/ART-73712",
    "FileSize": "6.7 MiB",
    "FileModifyDate": "2022:04:25 01:57:44+05:30",
    "FileAccessDate": "2022:04:25 01:57:44+05:30",
    "FileCreateDate": "2022:04:25 01:57:44+05:30",
    "FilePermissions": "-rw-rw-rw-"
  },
  "File": {
    "FileType": "DNG",
    "FileTypeExtension": "dng",
    "MIMEType": "image/x-adobe-dng",
    "ExifByteOrder": "Little-endian (Intel, II)"
  },
  "EXIF:IFD0": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 256,
    "ImageHeight": 164,
    "BitsPerSample": 8,
    "Compression": "Uncompressed",
    "PhotometricInterpretation": "BlackIsZero",
    "Make": "Canon",
    "Model": "Canon EOS 350D DIGITAL",
    "StripOffsets": 90842,
    "Orientation": "Horizontal (normal)",
    "SamplesPerPixel": 1,
    "RowsPerStrip": 164,
    "StripByteCounts": 41984,
    "PlanarConfiguration": "Chunky",
    "Software": "Adobe Photoshop Lightroom Classic 11.0.1 (Windows)",
    "ModifyDate": "2022:04:21 10:40:58",
    "Artist": "unknown",
    "ImageNumber": 176,
    "DNGVersion": "1.4.0.0",
    "DNGBackwardVersion": "1.1.0.0",
    "UniqueCameraModel": "Canon EOS 350D",
    "LocalizedCameraModel": "Canon EOS 350D Digital",
    "ColorMatrix1": "0.616 -0.0945 -0.0745 -0.6847 1.3563 0.3684 -0.0802 0.1086 0.7554",
    "ColorMatrix2": "0.6019 -0.0617 -0.0966 -0.8645 1.5882 0.2975 -0.153 0.1719 0.7642",
    "CameraCalibration1": "1.0491 0 0 0 1 0 0 0 0.9973",
    "CameraCalibration2": "1.0491 0 0 0 1 0 0 0 0.9973",
    "AnalogBalance": "1 1 1",
    "AsShotNeutral": "0.444396 1 0.669514",
    "BaselineExposure": 0.25,
    "BaselineNoise": 1.33,
    "BaselineSharpness": 1.5,
    "LinearResponseLimit": 1,
    "CameraSerialNumber": 830550471,
    "DNGLensInfo": "10-22mm f/?",
    "ShadowScale": 1,
    "CalibrationIlluminant1": "Standard Light A",
    "CalibrationIlluminant2": "D65",
    "RawDataUniqueID": "157A7EF208EDE0942DE6A8D6C0D05FB0",
    "OriginalRawFileName": "_MG_2224.CR2",
    "CameraCalibrationSig": "com.adobe",
    "ProfileCalibrationSig": "com.adobe",
    "ProfileName": "Adobe Standard",
    "ProfileEmbedPolicy": "Allow Copying",
    "ProfileCopyright": "Copyright 2008 Adobe Systems, Inc.",
    "ForwardMatrix1": "0.7622 0.0784 0.1237 0.3986 0.7409 -0.1395 0.0431 -0.1074 0.8894",
    "ForwardMatrix2": "0.7622 0.0784 0.1237 0.3986 0.7409 -0.1395 0.0431 -0.1074 0.8894",
    "PreviewApplicationName": "Adobe Photoshop Lightroom Classic",
    "PreviewApplicationVersion": "11.0.1",
    "PreviewSettingsDigest": "585cde2233e520f7eb7e9846bbb08b71",
    "PreviewColorSpace": "Gray Gamma 2.2",
    "PreviewDateTime": "2022:04:21 10:40:57+04:00",
    "ProfileLookTableDims": "36 8 16",
    "ProfileLookTableData": "(large array of 13824 float values)",
    "NoiseProfile": "2.40802624551766e-005 2.60408067131826e-007",
    "NewRawImageDigest": "fa5345e9e74f9ed35749b4eb34b893bc",
    "ThumbnailTIFF": "(Binary data 42194 bytes, use -b option to extract)"
  },
  "EXIF:SubIFD": {
    "SubfileType": "Full-resolution image",
    "ImageWidth": 3516,
    "ImageHeight": 2328,
    "BitsPerSample": 16,
    "Compression": "JPEG",
    "PhotometricInterpretation": "Color Filter Array",
    "SamplesPerPixel": 1,
    "PlanarConfiguration": "Chunky",
    "TileWidth": 256,
    "TileLength": 240,
    "TileOffsets": "(Binary data 1117 bytes, use -b option to extract)",
    "TileByteCounts": "(Binary data 839 bytes, use -b option to extract)",
    "CFARepeatPatternDim": "2 2",
    "CFAPattern2": "0 1 1 2",
    "CFAPlaneColor": "Red,Green,Blue",
    "CFALayout": "Rectangular",
    "BlackLevelRepeatDim": "1 1",
    "BlackLevel": 0,
    "BlackLevelDeltaV": "(large array of 2314 rational64s values)",
    "WhiteLevel": 4095,
    "DefaultScale": "1 1",
    "DefaultCropOrigin": "10 5",
    "DefaultCropSize": "3456 2304",
    "BayerGreenSplit": 0,
    "AntiAliasStrength": 1,
    "BestQualityScale": 1,
    "ActiveArea": "14 42 2328 3516",
    "MaskedAreas": "14 0 2328 40",
    "NoiseProfile": "2.40802624551766e-005 2.60408067131826e-007"
  },
  "EXIF:SubIFD1": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 1024,
    "ImageHeight": 655,
    "BitsPerSample": 8,
    "Compression": "JPEG",
    "PhotometricInterpretation": "BlackIsZero",
    "PreviewImageStart": 132826,
    "SamplesPerPixel": 1,
    "RowsPerStrip": 655,
    "PreviewImageLength": 75480,
    "PlanarConfiguration": "Chunky",
    "PreviewApplicationName": "Adobe Photoshop Lightroom Classic",
    "PreviewApplicationVersion": "11.0.1",
    "PreviewSettingsDigest": "585cde2233e520f7eb7e9846bbb08b71",
    "PreviewColorSpace": "Gray Gamma 2.2",
    "PreviewDateTime": "2022:04:21 10:40:57+04:00",
    "PreviewImage": "(Binary data 75480 bytes, use -b option to extract)"
  },
  "EXIF:SubIFD2": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 2048,
    "ImageHeight": 1365,
    "BitsPerSample": "8 8 8",
    "Compression": "Lossy JPEG",
    "PhotometricInterpretation": "Linear Raw",
    "SamplesPerPixel": 3,
    "PlanarConfiguration": "Chunky",
    "TileWidth": 512,
    "TileLength": 464,
    "TileOffsets": "(Binary data 83 bytes, use -b option to extract)",
    "TileByteCounts": "(Binary data 71 bytes, use -b option to extract)",
    "BlackLevel": "22 16 19",
    "PreviewApplicationName": "Adobe Photoshop Lightroom Classic",
    "PreviewApplicationVersion": "11.0.1",
    "OpcodeList2": "MapPolynomial, MapPolynomial, MapPolynomial",
    "CacheVersion": "0.1.0.0"
  },
  "EXIF:SubIFD3": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 512,
    "ImageHeight": 341,
    "BitsPerSample": "8 8 8",
    "Compression": "Lossy JPEG",
    "PhotometricInterpretation": "Linear Raw",
    "SamplesPerPixel": 3,
    "PlanarConfiguration": "Chunky",
    "TileWidth": 512,
    "TileLength": 341,
    "TileOffsets": 867576,
    "TileByteCounts": 43074,
    "BlackLevel": "31 37 32",
    "PreviewApplicationName": "Adobe Photoshop Lightroom Classic",
    "PreviewApplicationVersion": "11.0.1",
    "OpcodeList2": "MapPolynomial, MapPolynomial, MapPolynomial",
    "CacheVersion": "0.1.0.0"
  },
  "EXIF:SubIFD4": {
    "SubfileType": "Reduced-resolution image",
    "ImageWidth": 256,
    "ImageHeight": 171,
    "BitsPerSample": "8 8 8",
    "Compression": "Lossy JPEG",
    "PhotometricInterpretation": "Linear Raw",
    "SamplesPerPixel": 3,
    "PlanarConfiguration": "Chunky",
    "TileWidth": 256,
    "TileLength": 171,
    "TileOffsets": 910650,
    "TileByteCounts": 12028,
    "BlackLevel": "31 37 32",
    "PreviewApplicationName": "Adobe Photoshop Lightroom Classic",
    "PreviewApplicationVersion": "11.0.1",
    "OpcodeList2": "MapPolynomial, MapPolynomial, MapPolynomial",
    "CacheVersion": "0.1.0.0"
  },
  "XMP:XMP-x": {
    "XMPToolkit": "Adobe XMP Core 7.0-c000 1.000000, 0000/00/00-00:00:00        "
  },
  "XMP:XMP-dc": {
    "Format": "image/dng",
    "Creator": ["unknown"]
  },
  "XMP:XMP-aux": {
    "SerialNumber": 830550471,
    "LensInfo": "10-22mm f/?",
    "Lens": "10.0-22.0 mm",
    "ImageNumber": 176
  },
  "XMP:XMP-xmp": {
    "ModifyDate": "2022:04:21 10:40:58+04:00",
    "CreateDate": "2008:12:14 15:54:54",
    "CreatorTool": "Adobe Photoshop Lightroom Classic 11.0.1 (Windows)",
    "MetadataDate": "2022:04:21 10:40:58+04:00"
  },
  "XMP:XMP-photoshop": {
    "DateCreated": "2008:12:14 15:54:54-04:00"
  },
  "XMP:XMP-xmpMM": {
    "DocumentID": "xmp.did:e297762f-f029-7140-bb5f-45c3972a1c81",
    "PreservedFileName": "sample1.dng",
    "OriginalDocumentID": "157A7EF208EDE0942DE6A8D6C0D05FB0",
    "InstanceID": "xmp.iid:e297762f-f029-7140-bb5f-45c3972a1c81",
    "History": [{
      "Action": "derived",
      "Parameters": "saved to new location"
    },{
      "Action": "saved",
      "Changed": "/",
      "InstanceID": "xmp.iid:e297762f-f029-7140-bb5f-45c3972a1c81",
      "SoftwareAgent": "Adobe Photoshop Lightroom Classic 11.0.1 (Windows)",
      "When": "2022:04:21 10:40:58+04:00"
    }],
    "DerivedFrom": {
      "DocumentID": "157A7EF208EDE0942DE6A8D6C0D05FB0",
      "OriginalDocumentID": "157A7EF208EDE0942DE6A8D6C0D05FB0"
    }
  },
  "XMP:XMP-crd": {
    "CameraProfile": "Camera Standard",
    "LookName": ""
  },
  "XMP:XMP-crs": {
    "Version": 14.0,
    "ProcessVersion": 11.0,
    "WhiteBalance": "As Shot",
    "ColorTemperature": 5200,
    "Tint": "+12",
    "Exposure2012": 0.00,
    "Contrast2012": 0,
    "Highlights2012": 0,
    "Shadows2012": 0,
    "Whites2012": 0,
    "Blacks2012": 0,
    "Texture": 0,
    "Clarity2012": 0,
    "Dehaze": 0,
    "ParametricShadows": 0,
    "ParametricDarks": 0,
    "ParametricLights": 0,
    "ParametricHighlights": 0,
    "ParametricShadowSplit": 25,
    "ParametricMidtoneSplit": 50,
    "ParametricHighlightSplit": 75,
    "Sharpness": 40,
    "SharpenRadius": "+1.0",
    "SharpenDetail": 25,
    "SharpenEdgeMasking": 0,
    "LuminanceSmoothing": 0,
    "ColorNoiseReduction": 25,
    "ColorNoiseReductionDetail": 50,
    "ColorNoiseReductionSmoothness": 50,
    "GrayMixerRed": 0,
    "GrayMixerOrange": 0,
    "GrayMixerYellow": 0,
    "GrayMixerGreen": 0,
    "GrayMixerAqua": 0,
    "GrayMixerBlue": 0,
    "GrayMixerPurple": 0,
    "GrayMixerMagenta": 0,
    "SplitToningShadowHue": 0,
    "SplitToningShadowSaturation": 0,
    "SplitToningHighlightHue": 0,
    "SplitToningHighlightSaturation": 0,
    "SplitToningBalance": 0,
    "ColorGradeMidtoneHue": 0,
    "ColorGradeMidtoneSat": 0,
    "ColorGradeShadowLum": 0,
    "ColorGradeMidtoneLum": 0,
    "ColorGradeHighlightLum": 0,
    "ColorGradeBlending": 50,
    "ColorGradeGlobalHue": 0,
    "ColorGradeGlobalSat": 0,
    "ColorGradeGlobalLum": 0,
    "AutoLateralCA": 0,
    "LensProfileEnable": 0,
    "LensManualDistortionAmount": 0,
    "VignetteAmount": 0,
    "PerspectiveUpright": 0,
    "PerspectiveVertical": 0,
    "PerspectiveHorizontal": 0,
    "PerspectiveRotate": 0.0,
    "PerspectiveAspect": 0,
    "PerspectiveScale": 100,
    "PerspectiveX": 0.00,
    "PerspectiveY": 0.00,
    "GrainAmount": 0,
    "PostCropVignetteAmount": 0,
    "ShadowTint": 0,
    "RedHue": 0,
    "RedSaturation": 0,
    "GreenHue": 0,
    "GreenSaturation": 0,
    "BlueHue": 0,
    "BlueSaturation": 0,
    "OverrideLookVignette": false,
    "ToneCurveName2012": "Linear",
    "CameraProfile": "Adobe Standard",
    "CameraProfileDigest": "076EB93F90F85BD4B9296655584B5D05",
    "ToggleStyleDigest": "B5217C8386FEFBEB46D41DB1B1AFF8EC",
    "ToggleStyleAmount": 1,
    "HasSettings": true,
    "CropTop": 0.074189,
    "CropLeft": 0.056643,
    "CropBottom": 0.64915,
    "CropRight": 0.656025,
    "CropAngle": 0,
    "CropConstrainToWarp": 0,
    "HasCrop": true,
    "AlreadyApplied": false,
    "ToneCurvePV2012": ["0, 0","255, 255"],
    "ToneCurvePV2012Red": ["0, 0","255, 255"],
    "ToneCurvePV2012Green": ["0, 0","255, 255"],
    "ToneCurvePV2012Blue": ["0, 0","255, 255"],
    "Look": {
      "Amount": 1,
      "Group": "Profiles",
      "Name": "Adobe Monochrome",
      "Parameters": {
        "CameraProfile": "Adobe Standard",
        "Clarity2012": "+8",
        "ConvertToGrayscale": true,
        "LookTable": "73ED6C18DDE909DD7EA2D771F5AC282D",
        "ProcessVersion": 11.0,
        "ToneCurvePV2012": ["0, 0","64, 56","128, 128","192, 197","255, 255"],
        "ToneCurvePV2012Blue": ["0, 0","255, 255"],
        "ToneCurvePV2012Green": ["0, 0","255, 255"],
        "ToneCurvePV2012Red": ["0, 0","255, 255"],
        "Version": 14.0
      },
      "SupportsAmount": false,
      "SupportsMonochrome": false,
      "SupportsOutputReferred": false,
      "UUID": "0CFE8F8AB5F63B2A73CE0B0077D20817"
    }
  },
  "EXIF:ExifIFD": {
    "ExposureTime": "1/800",
    "FNumber": 7.1,
    "ExposureProgram": "Manual",
    "ISO": 100,
    "ExifVersion": "0231",
    "DateTimeOriginal": "2008:12:14 15:54:54",
    "CreateDate": "2008:12:14 15:54:54",
    "OffsetTime": "+04:00",
    "OffsetTimeOriginal": "-04:00",
    "ShutterSpeedValue": "1/800",
    "ApertureValue": 7.1,
    "ExposureCompensation": 0,
    "MaxApertureValue": 4.0,
    "MeteringMode": "Multi-segment",
    "Flash": "Off, Did not fire",
    "FocalLength": "20.0 mm",
    "ColorSpace": "Uncalibrated",
    "CustomRendered": "Normal",
    "ExposureMode": "Auto bracket",
    "WhiteBalance": "Auto",
    "SceneCaptureType": "Standard",
    "SerialNumber": 830550471,
    "LensInfo": "10-22mm f/?",
    "LensModel": "10.0-22.0 mm"
  },
  "EXIF:ProfileIFD": {
    "ColorMatrix1": "0.6159 -0.0945 -0.0745 -0.6846 1.3563 0.3684 -0.0802 0.1086 0.7555",
    "ColorMatrix2": "0.6018 -0.0617 -0.0965 -0.8645 1.5881 0.2975 -0.153 0.1719 0.7642",
    "CalibrationIlluminant1": "Standard Light A",
    "CalibrationIlluminant2": "D65",
    "ProfileCalibrationSig": "com.adobe",
    "ProfileName": "ACR 4.4",
    "ProfileEmbedPolicy": "Allow Copying",
    "ProfileCopyright": "Copyright 2008 Adobe Systems, Inc.",
    "ForwardMatrix1": "0.8255 -0.0022 0.1411 0.424 0.6455 -0.0695 0.0908 -0.234 0.9683",
    "ForwardMatrix2": "0.7623 0.0785 0.1235 0.3987 0.7409 -0.1396 0.0431 -0.1074 0.8894"
  },
  "Composite": {
    "Aperture": 7.1,
    "CFAPattern": "[Red,Green][Green,Blue]",
    "ImageSize": "3516x2328",
    "Megapixels": 8.2,
    "ShutterSpeed": "1/800",
    "SubSecDateTimeOriginal": "2008:12:14 15:54:54-04:00",
    "SubSecModifyDate": "2022:04:21 10:40:58+04:00",
    "FocalLength35efl": "20.0 mm",
    "LightValue": 15.3,
    "LensID": "10.0-22.0mm"
  }
}]

**Reply by SAN2INGIT** [reply]
[Uploading sample1crop.zip…]()

**Comment by SAN2INGIT** [comment]
My expectation from this forum is to get a command that would generate a proper thumbnail for both the original and the cropped version of the dng file. 
The generated thumbnail should be having a monochrome and cropped image if the main image is monochromatic and cropped.

**Comment by snibgo** [comment]
So, exiftool says:

sample1original.dng:
```
"EXIF:SubIFD": {
"SubfileType": "Full-resolution image",
"ImageWidth": 3516,
"ImageHeight": 2328,
```

sample1crop.dng:
```
"EXIF:SubIFD": {
"SubfileType": "Full-resolution image",
"ImageWidth": 3516,
"ImageHeight": 2328,
```
And they are both encoded as RGB Color Filter Arrays. Because they are DNG files, IM has de-Bayered the raw images.

So, I ask again, what tool made the "cropped" version? How do you know it is cropped?

Both files contain thumbnail images. These may be grayscale, and may be cropped. IM won't extract them. If you want them, you can use exiftool, with the "-b" option.

IM can, of course, make an image monochrome and cropped, eg:
```
magick.exe C:\sample1crop.dng -colorspace Gray -crop 100x600+50+60 -resize 500x375 C:\x.JPG
```

**Reply by SAN2INGIT** [reply]
Well, both these files were handed over to us from the client, so not sure how they generated the cropped file, but in open eyes it look like one image is a cropped version of the other one, as I am not able to upload the dng file, trying to upload the png version of both the files for better understanding:
Cropped image:
![image](https://user-images.githubusercontent.com/32328926/165283632-8e2c73f4-4e19-4f34-b761-a9dcac1b3d47.png)

Original image:
![image](https://user-images.githubusercontent.com/32328926/165287015-3e7889a1-7d7a-41d7-beea-9c44b7e4c5ff.png)

Our use case is, we will need to apply the same cmd to different images to generate the thumbnail, so can't really apply a crop or monochrome cmd on a particular image before generating the thumbnail image.

**Comment by snibgo** [comment]
> ... but in open eyes it look like one image is a cropped version ...

Using what software?

> ... trying to upload the png version of both the files ...

Yes, I can see the cropped PNG version is cropped, and monochrome. Both images have had JPEG compression.

I suppose you made the PNG files from the DNG files. How did you make the PNG files? ImageMagick, or what tool?

The two PNG files seem to be slightly smaller versions of thumbnails that are in the DNG files.

```
f:\web\im>%IMG7%magick t_cropped.png info:
t_cropped.png PNG 1023x655 1023x655+0+0 8-bit sRGB 510544B 0.000u 0:00.002

f:\web\im>%IMG7%magick t_orig.png info:
t_orig.png PNG 1023x682 1023x682+0+0 8-bit sRGB 821160B 0.000u 0:00.002
```
From exiftool:
```
"EXIF:SubIFD1": {
"SubfileType": "Reduced-resolution image",
"ImageWidth": 1024,
"ImageHeight": 683,
```

```
"EXIF:SubIFD1": {
"SubfileType": "Reduced-resolution image",
"ImageWidth": 1024,
"ImageHeight": 655,
```
As I said, IM will de-Bayer the raw images. If instead you want to extract these thumbnails, use exiftool.

**Reply by SAN2INGIT** [reply]
Using what software? _Will enquire and let you know_
How did you make the PNG files? ImageMagick, or what tool? _I just opened in snap&sketch and copied the images._


Actually, I tried ExifTool as well, getting error: _Warning: [Minor] Not decoding some large array(s). Ignore minor errors to decode -_ 
Of course it's not the correct forum to discuss about it.
[Uploading sample1crop.zip…]()

**Comment by snibgo** [comment]
I have never heard of "snap&sketch". I suspect it is extracting the thumbnail that is embedded in the DNG file.

You keep posting a link, that is simply a link to this discussion, not a zip file.

**Reply by SAN2INGIT** [reply]
That link is actually the result of a failed attempt to upload the .dng file zip. Not sure what's wrong though.
_Snip & Sketch_ is the replacement of snipping tool in Windows 11. Apologies for my typo, where I mentioned 'Snap & Sketch'. 
I don't think it's the embedded thumbnail, I just opened the dng file with the app.

**Comment by snibgo** [comment]
> I don't think it's the embedded thumbnail, I just opened the dng file with the app.

How large is the image from "Snip & Sketch"? As I showed above, the full-resolution image is 3516x2328 pixels. If the image from "Snip & Sketch" is much smaller, then it is reading the embedded thumbnail.

That is common practise for two reasons. (1) Reading the embedded JPEG thumbnail is much faster than de-Bayering the raw sensor data. (2) The embedded JPEG has baked-in the processing the photographer told the camera to do, such as "make the image monochrome, and crop it".

**Reply by SAN2INGIT** [reply]
Thanks, @snibgo for your time, and sorry for the delay in response.
Since I have not been able to upload the dng files in this forum, I uploaded the same in hightail and created a link: https://spaces.hightail.com/space/ThDEDYZVey 
Can you please download the DNG files from there?

The primary goal for me is to generate a thumbnail, which I am not getting appropriately on applying the ImageMagick commands. ExifTool also getting some error as well.

Also, the software used to generate the cropped monochrome dng file is Adobe Lightroom Classic for modifying/cropping the images.

**Comment by snibgo** [comment]
I can't download those files, perhaps because I won't accept their cookies.

**Comment by SAN2INGIT** [comment]
I think our understanding and discussion went a bit off track, I analyzed and understood the problem is:
Whether Imagemagick is able to obey the metadata: EXIF:IFD0:PreviewColorSpace and XMP:XMP-crs:crop properties  for DNG files when resizing the same.
With the cmds that I used, IM was not able to obey those.

I am closing this thread and open a new thread to initiate the discussion on that front.
